src/plugins/autonomy/ArduPilot/ArduPilot.cpp

```diff
@@ -146,7 +146,7 @@ void ArduPilot::start_receive() {
                     ba::placeholders::bytes_transferred));
 }
 
-void ArduPilot::close(double t) {
+void ArduPilot::close(const double &t) {
```

Comment:
Is there a need to add a reference here? This will break the api for downstream plugins. In particular, if they are not using ``override`` they won't know about the change and ``close`` would just not be called.

Reply:
Ok. We have to figure out when we will update the step(), init(), and close() functions to better follow const correctness.